Changing makecert example to make the dates match

rjmiller-adot opened the pull request:
Changed the dates to mm/dd/yyyy format and also made the caption match the date in the example.

azurecla commented:
Hi __@bcraun__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

deneha commented:
@bcraun Thanks for the contribution!

rjmiller-adot committed:
Changing makecert example to make the dates match

Changed the dates to mm/dd/yyyy format and also made the caption match the date in the example.
